Rakefile

```diff
@@ -3,33 +3,31 @@ require 'yaml'
 require 'logger'
 require 'cucumber/rake/task'
 require 'sqlite3'
+require 'zlib'
 
-if ENV.has_key? 'ENVIRONMENT'
-  ENV['ENVIRONMENT'] = 'test'
-else
-  ENV['ENVIRONMENT'] = 'development'
-end
+ENV['ENVIRONMENT'] ||= 'development'
```

Comment:
@mvemjsun I had to do this because otherwise I couldn't set the `ENVIRONMENT` variable to other rather than `test` or `development`, but I wanted to use development_docker